src/connector/src/parser/protobuf/parser.rs

```diff
@@ -494,12 +494,16 @@ pub fn from_protobuf_value(
             ScalarImpl::List(ListValue::new(rw_values))
         }
         Value::Bytes(value) => ScalarImpl::Bytea(value.to_vec().into_boxed_slice()),
-        _ => {
-            let err_msg = format!(
-                "protobuf parse error.unsupported type {:?}, value {:?}",
-                field_desc, value
-            );
-            return Err(RwError::from(InternalError(err_msg)));
+        Value::Map(map) => {
+            let mut rw_values = Vec::with_capacity(map.len());
+            for (k, v) in map {
```

Comment:
is it guaranteed that the `iter` follows the same order?

Reply:
Good catch. `map` is a `HashMap` so the iteration order will not be deterministic.